scripts/build_windows_fluent_bit.ps1

```diff
@@ -283,8 +283,8 @@ Compress-Archive -Path "${StagingDirectory}\*" -Destination "${AWSForFluentBitRo
 # Compress ecs_windows_forward_daemon folder which needs to be added to the image
 Compress-Archive -Path "${AWSForFluentBitRootDir}\ecs_windows_forward_daemon\*" -Destination "${AWSForFluentBitRootDir}\build\windows\ecs_windows_forward_daemon.zip"
 
-# Copy the version of aws-for-fluent-bit
-Copy-Item -Path "${AWSForFluentBitRootDir}\AWS_FOR_FLUENT_BIT_VERSION" -Destination "${AWSForFluentBitRootDir}\build\windows\AWS_FOR_FLUENT_BIT_VERSION"
+# The version of aws-for-fluent-bit is set in each image from the env var, as the pipeline updates multiple
+# images found in the windows.versions file
```

Comment:
@rawahars agreed on slack its safe to remove this. From my review of the code, the AWS_FOR_FLUENT_BIT_VERSION isn't actually used. This is for the reason explained in the comment, the windows.versions file controls windows versions and this is used to set and env var for each build.